scripts/release/release.js

```diff
@@ -0,0 +1,10 @@
+/* eslint import/no-extraneous-dependencies: ["error", {"devDependencies": true}] */
+const shell = require('shelljs');
```

Comment:
I'm not sure if this is a concern here, but thought I'd mention it since @bjankord ran into this issue with terra-toolkit: https://github.com/cerner/terra-toolkit/issues/10

Reply:
Good to know. This is probably an issue here too. I copied the release script from terra-toolkit. I'll look into it.

Reply:
This isn't an issue here because I'm just checking in the package-lock.json file. According to NPM docs, it's intended to be checked into source control. https://docs.npmjs.com/files/package-lock.json
¯\_(ツ)_/¯